### ISSUE TITLE
Missing parent: GO:0004583 dolichyl-phosphate-glucose-glycolipid alpha-glucosyltransferase activity subclasses

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#28029]
* **GO term ID and label for which you request a new superclass**

It seems that, based on definition and annotations

GO:0106073  dolichyl pyrophosphate Glc2Man9GlcNAc2 alpha-1,2-glucosyltransferase activity
Catalysis of the addition of the third glucose residue to the lipid-linked oligosaccharide precursor for N-linked glycosylation; the transfer of glucose from dolichyl phosphate glucose (Dol-P-Glc) on to the lipid-linked oligosaccharide Glc(2)Man(9)GlcNAc(2)-PP-Dol. PMID:9597543

and

GO:0042281 | dolichyl pyrophosphate Man9GlcNAc2 alpha-1,3-glucosyltransferase activity
Catalysis of the addition of the first glucose residue to the lipid-linked oligosaccharide precursor for N-linked glycosylation; the transfer of glucose from dolichyl phosphate glucose (Dol-P-Glc) on to the lipid-linked oligosaccharide Man(9)GlcNAc(2)-PP-Dol.

and

GO:0042283 | dolichyl pyrophosphate Glc1Man9GlcNAc2 alpha-1,3-glucosyltransferase activity
Catalysis of the addition of the second glucose residue to the lipid-linked oligosaccharide precursor for N-linked glycosylation; the transfer of glucose from dolichyl phosphate glucose (Dol-P-Glc) on to the lipid-linked oligosaccharide Glc(1)Man(9)GlcNAc(2)-PP-Dol. [[PMID:12480927](http://www.ncbi.nlm.nih.gov/pubmed/12480927)]

should be is_a 
GO:0004583 dolichyl-phosphate-glucose-glycolipid alpha-glucosyltransferase activity
Catalysis of the transfer of an alpha-D-glucosyl residue from dolichyl-phosphate D-glucose into a membrane lipid-linked oligosaccharide.

* **New superclass (parent) suggested**

* **Reference(s), if appropriate**
PMID:nnnnnnn

**Comment by pgaudet** [comment]
Feedback from Ben Kellman (expect contacted by Frederique Lisacek): 


I am not an experienced curator, so I can't comment with certainty on the meaning of parentage. However, I have spent time in glycan biosynthesis, so I may be of help there.

Since you work in yeast, you are likely aware of ALG genes. For citations, PMID:11425794 provides some useful citations in the broader context of Dolichol-P-P-GlcNAc2Man9Glc3 biosynthesis. The final paragraph references the yeast knockouts that uncovered the alg genes responsible for the process.

Here is a more modern figure summarizing the process and context:
image.png
Essentials of Glycobiology [Internet]. 4th edition. Cold Spring Harbor (NY)

I'm pointing at the whole process because when I think of the dolichol-P-glycan precursor to N-glycosylation, I rarely think specifically of the terminal tri-glucose additions. Over a dozen enzymes are responsible for building N-glycan precursor. That said, these three enzymes are similar in that they share a precursor, glucose.

In summary, I think your proposal makes sense due to the similarity of these reactions. And, provided some structure already exists surrounding the context of these three reactions. E.g.: 
N-glycan biosynthesis
N-glycan precursor biosynthesis
dol-P synthesis
cytosolic 
glcnac additions
DPAGT1, 13/14
mannose additions
ALG1, 2, 2, 11, 11
in ER 
flip dol-PP-glycan into ER
mannose additions
ALG3, ALG9, ALG12, ALG9
glucose additions
ALG6, ALG8, ALG10
dolichol to protein transfer ....

Hope this is helpful,
Ben

**Comment by kiyoko** [comment]
I agree with the suggestion to create a new superclass for these three genes.

**Comment by ValWood** [comment]
Thanks both I will add!

**Comment by sjm41** [comment]
@raymond91125  Could you take care of the proposed obsoletion please?

**Comment by raymond91125** [comment]
@sjm41 Obsoleted GO:0033556. You may close the ticket if all is done. Thanks.
